woocommerce/google-listings-and-ads
# UX improvement on WordPress.com account connection

### Changes proposed in this Pull Request:

Part of #1065 .

This PR changes the WordPress.com account connection card to the newer design. The account connection functionality and track events should all remain the same.

### Screenshots:

To be connected: 

![image](https://user-images.githubusercontent.com/417342/139316928-9e47e17d-7877-474a-a46d-e08f4c2cebf5.png)

Connected account: 

![image](https://user-images.githubusercontent.com/417342/139316866-b7b0aa97-6e9c-4429-a989-ed6cf89d5f94.png)

### Detailed test instructions:

1. Disconnect WordPress.com account in connection test page.
2. Go to Setup MC flow. You should see the first screenshot above.
3. Connect your WordPress.com account.
4. When connection is done, you should see the second screenshot above.



### Changelog entry

> Update - Update the WordPress.com account connection UI to the newer design.
